runtime/runtime/Cargo.toml

```diff
@@ -33,11 +33,11 @@ near-vm-runner = { path = "../../runtime/near-vm-runner" }
 near-vm-errors = { path = "../../runtime/near-vm-errors" }
 
 [features]
-default = []
+default = ["cpu_compatibility_checks"]
```

Comment:
Does this do anything? we depend on `vm-runner` with default features anyway, no?